cmd/prometheus/main.go

```diff
@@ -136,6 +137,9 @@ func main() {
 	a.Flag("config.file", "Prometheus configuration file path.").
 		Default("prometheus.yml").StringVar(&cfg.configFile)
 
+	a.Flag("autoreload.config", "Automatically reload config file on changes").
```

Comment:
I think the convention is `config.autoreload` (see the line above)

Reply:
done